### PR TITLE
Enhance role mapping modal based on new design

### PR DESCRIPTION
**Pull request by ajaypratap003**
https://issues.redhat.com/browse/MGDSTRM-6396

Create role mapping:
image

image

image

Edit role mapping:
image

### CODE REVIEW

File: ui/src/app/pages/roles/components/modals/grantAccessModal.tsx [hunk 1]
```diff
@@ -320,7 +347,7 @@ export class GrantAccessModal extends PureComponent<GrantAccessModalProps, Grant
```
Comment (jgiardino) [inline ui/src/app/pages/roles/components/modals/grantAccessModal.tsx:347]: While working on updates for this, one of our content designers provided a new description to make the text more consistent with other text we're using for the button label and modal header.

Can you replace the following text:
> Manage access to resources in this Service Registry instance by assigning permissions to an account.
With this text:
> Grant access to resources in this Service Registry instance by assigning a role to an account.

Can you also replace this text:
> Manage access to resources in the Registry by assigning permissions to an account.
With this text:
> Grant access to resources in the Registry by assigning permissions to an account

**Comment by jgiardino** [comment]
Your examples look great! 

There's one thing missing in the "Edit role mapping" example that you show. The selected account's current role assignment should be indicated in the radio group, similar to the third example you show for Create. For example, if the current role assignment is "Manager" then the radio button label should be "Manager (current role)".

Since I can't interact with these examples, I just want to mention that the "(current role)" label stays with that radio button if the user chooses a different radio button. I.e. it doesn't move to "Viewer" if I select that option to change the role assignment.

I also posted a comment inline about some text string changes that would be great to include as well.

File: ui/src/app/pages/roles/components/modals/grantAccessModal.tsx (hunk 1 above)
Comment (ajaypratap003) [reply ui/src/app/pages/roles/components/modals/grantAccessModal.tsx:348]: done

**Comment by ajaypratap003** [comment]
> Your examples look great!
> 
> There's one thing missing in the "Edit role mapping" example that you show. The selected account's current role assignment should be indicated in the radio group, similar to the third example you show for Create. For example, if the current role assignment is "Manager" then the radio button label should be "Manager (current role)".
> 
> Since I can't interact with these examples, I just want to mention that the "(current role)" label stays with that radio button if the user chooses a different radio button. I.e. it doesn't move to "Viewer" if I select that option to change the role assignment.
> 
> I also posted a comment inline about some text string changes that would be great to include as well.

@jgiardino  Thanks for the feedback. I've updated strings as you suggested and added current role for edit. 
Current role string will not shift while changing radio options.

**Comment by EricWittmann** [comment]
Looks really good, thanks @ajaypratap003 !

**Pull request merged**